src/views/Points.js

```diff
@@ -88,6 +88,79 @@ export default function Points() {
     addRejectedPoint,
   ] = useRejectedIncomingPointTransfers();
 
+  const maybeOutgoingPoints = useMemo(() => {
+    return controlledPoints.matchWith({
```

Comment:
nit: just use an inline function `() => controlledPoints.matchWith(...)`

another nit: since we're returning a `Maybe` from a `Maybe` we can use `.chain` to only operate on the `Just` value like `controlledPoints.chain(value => values.matchWith(...))`

Reply:
we're actually doing that twice, also on like 98, so `c.value.ownedPoints.map(point => getDetails(point).map(details => ({point, hasTransferProxy(details)}))` would be cleaner